app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallViewModel.kt

```diff
@@ -48,7 +49,6 @@ class InitiatingCallViewModel @Inject constructor(
 
     init {
         viewModelScope.launch {
-            launch { initiateCall() }
             launch { initializeScreenState() }
```

Comment:
Since statement in a coroutine runs in sequential, we can do this
```
launch { 
    initializeScreenState() 
    initiateCall()
}
```
and keep `initializeScreenState()` only  responsible for initializing the screen

We can do it in this way also:
```
val job = launch { initiateCall() }
job.join()
launch { initiateCall() }
launch { observeStartedCall() }
```

Reply:
Due to you already having it solved in another draft I will then close this PR!  Great work on finding it as well hehe